### PR TITLE
refactor: remove unused oracle addresses

### PR DESCRIPTION
**Pull request by manu0466**
## Overview

This PR removes all references to the old oracles from the messages and the contract config.   
Additionally, to properly remove the old oracle addresses from the config, the migration logic has been updated and
the `MsgMigrate` has been converted to an enum to support future migration logics.  

To migrate the contract from the version `v0.4.18` the following migrate message must be sent: 
```json
{
  "v0_4_18_to_v0_4_19": {}
}
```

closes: MILKTIA-32

## What changes have been made in this PR?

- [ ] Removed all the references to the old oracles
- [ ] Added a migration logic to remove the old oracles address from the state of the contract
- [ ] Refactored the migration logic to support future migrations

## Checklist

---

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 members = ["contracts/*", "packages/*"]
 
 [workspace.package]
-version = "0.4.18"
+version = "0.4.19"
```
Comment (manu0466) [inline Cargo.toml:5]: I have bumped this just to have an higher version and test everithing. Let me know if I should change it.

**Comment by manu0466** [comment]
Here some screenshots of the contract state before and after the migration.

## Contract state on version v0.4.18
0-4-18-version
old-contract-state

## Contract state on version v0.4.19
states-post-migration

File: contracts/staking/src/state.rs [hunk 2]
```diff
@@ -9,7 +9,6 @@ pub struct Config {
     pub native_token_denom: String,
     pub liquid_stake_token_denom: String,
     pub treasury_address: Addr,
-    pub operators: Option<Vec<Addr>>, //depr
```
Comment (manu0466) [inline contracts/staking/src/state.rs:12]: I have removed also this since was marked as deprecated

**Review by RiccardoM — APPROVED** [review]
Looks good to me!

**Pull request merged**